django/db/backends/ddl_references.py

```diff
@@ -198,3 +199,28 @@ def rename_column_references(self, table, old_column, new_column):
 
     def __str__(self):
         return self.template % self.parts
+
+
+class IndexExpressions(TableColumns):
```

Comment:
Could we maybe use a name that doesn't conflict with `django.db.models.indexes.IndexExpression`  since this one has about no knowledge about _indexes_ per-se.

Reply:
Agree! It was originally named `Expressions`,  it now makes sense to revert back to that.